srcpkgs/khelpcenter/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'khelpcenter'
 pkgname=khelpcenter
-version=17.12.2
+version=18.04.3
 revision=1
 build_style=cmake
 configure_args="-DBUILD_TESTING=OFF"
 hostmakedepends="extra-cmake-modules pkg-config"
 makedepends="kinit-devel khtml-devel grantlee5-devel xapian-core-devel"
 short_desc="KDE Help center"
 maintainer="Denis Revin <denis.revin@gmail.com>"
-license="GPL-2"
+license="GPL-2.0-only"
```

Comment:
*  This program is free software; you can redistribute it and/or modify
 *  it under the terms of the GNU General Public License as published by
 *  the Free Software Foundation; either version 2 of the License, or
 *  (at your option) any later version.

GPL-2.0-or-later